pkg/storage/tableland/store.go

```diff
@@ -60,17 +48,58 @@ var (
 			unique(rig_id, trait_type)
 		)`,
 	}
+	// LookupsDefinition defines the lookups table.
+	LookupsDefinition = TableDefinition{
+		Prefix: "lookups",
+		Schema: `(
+			renders_cid text,
+			layers_cid text,
+			image_full_name text,
+			image_full_alpha_name text,
+			image_medium_name text,
+			image_medium_alpha_name text,
+			image_thumb_name text,
+			image_thumb_alpha_name text,
+			animation_base_url text
+		)`,
+	}
+	// PilotSessionsDefinition defines the pilot sessions table.
+	PilotSessionsDefinition = TableDefinition{
+		Prefix: "pilot_sessions",
+		Schema: `(
+			id integer primary key,
+  		rig_id integer not null,
+  		pilot_contract text,
+  		pilot_id integer,
+  		start integer not null,
+  		end integer
+		)`,
+	}
```

Comment:
New `pilot_sessions` table. This is basically the same schema I sent @datadanne with the slight change that `start_time` and `end_time` are now `start` and `end`. I haven't heard that there is anything lacking with this schema, so I think we're good to go here, but let me know if anything should change @datadanne.

Reply:
Me and dtb have been discussing the schema a bit here: https://github.com/tablelandnetwork/rigs/commit/1d97b32ba315758d3ddc2a8a22a9bf4ece9bbf1b#r86016113

tldr: I think we should add a `owner text` column so that we can record who the rig owner was for every pilot sessions

Reply:
Cool, added it. Also switched back to `start_time` and `end_time` to avoid using the reserved word `end`.